canonical-web-and-design/versioned-static
# Remove cchardet; Rename package

We've been having significant problems with getting cchardet to install in production units.

I think I've found a way to remove any need for character encoding support, which will simplify things significantly.

I've also taken the opportunity to rename the package to put it in the `canonicalwebteam` namespace (along with upload-assets), which I think will be the new home for our python modules.

QA
--

Try this out with one of our sites in both python2 and python3. E.g. www.ubuntu.com:

``` bash
git clone -b new-versioned-static https://github.com/nottrobin/www.ubuntu.com  # Clone my branch
cd www.ubuntu.com
virtualenv env2       # Create python2 environment
python3 -m venv env3  # Create python3 environment
env2/bin/pip install -r requirements.txt  # Install python2 dependencies
env3/bin/pip install -r requirements.txt  # Install python3 dependencies
env2/bin/pip install ../  # Install this branch
env3/bin/pip install ../  # Install this branch
env2/bin/python manage.py runserver  # Run the site, python2. Go to http://127.0.0.1:8000
rm cache.sqlite  # Cache needs removing to avoid errors
env3/bin/python manage.py runserver  # Run the site, python3. Go to http://127.0.0.1:8000
```

After running the site with python2 and python3 respectively, browse around the site and check all the static files seem to load and they all have version string in their URL paths.
